helmctlr/helm_test.go

```diff
@@ -156,9 +185,12 @@ func TestResourceAddedHappyPathExists(t *testing.T) {
 		create:   2,
 		releases: 2,
 	}
+	updateCalled = false
 	assertCounters(t, ct, func() {
 		testController.ResourceAdded(newTestResource())
 		testController.ResourceAdded(newTestResource())
+
+		assert.True(t, updateCalled)
```

Comment:
can you not query the mock to find out if this happened somehow?

Reply:
Wasn't working, this feels more unit-testy.